docs/guide/dev/example_setups/README.md

```diff
@@ -36,7 +36,7 @@ The Article Editor is based on the [Standard package](https://ckeditor.com/ckedi
 
 ## Document Editor
 
-The Document Editor demo showcases a more robust editor designed for creating documents which are usually later printed or exported to PDF files using tools like [wkhtmltopdf](https://github.com/wkhtmltopdf/wkhtmltopdf) or [PhantomJS](https://github.com/ariya/phantomjs) (note: PhantomJS 2.x currently has a [known zoom issue](https://github.com/ariya/phantomjs/issues/13997)).
+The Document Editor demo showcases a more robust editor designed for creating documents which can later be {@link features/exportpdf/README exported to PDF} using the official [Export to PDF](https://ckeditor.com/cke4/addon/exportpdf) plugin.
```

Comment:
Good catch :+1: Have you checked entire docs for other things like this and places where we can link to PDF Export plugin too?

Reply:
Yes. There is also one place we could also change the editor API docs: https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins_print.html, but it would require a separate PR (as it's in `ckeditor4` repo). I can extract an issue if you want.

Reply:
Could you extract it to separate issue?

Reply:
ckeditor/ckeditor4#4077 - again I didn't know how mysterious should I be :see\_no\_evil: but after [this comment](https://github.com/ckeditor/ckeditor4/issues/4061#issuecomment-630714428) I guess my issue is OK.